backend2/src/core/article_parser.py

```diff
@@ -50,6 +51,27 @@ def url(self) -> str:
 
         return f"{self.PMC_BASE_URL}{self.article.external_id}"
 
+    def publish_date(self) -> date:
+        try:
+            year = int(self.tree.findall(".//front/article-meta/pub-date/year")[0].text)
+        except:
+            year = 2020
+        try:
+            month = int(self.tree.findall(".//front/article-meta/pub-date/month")[0].text)
+        except:
+            month = 1
+        try:
+            day = int(self.tree.findall(".//front/article-meta/pub-date/day")[0].text)
+        except:
+            day = 1
```

Comment:
I think it may be better to fall back to current year, current month and current day assuming that most articles get fetched at the day of publishing.